### DISCUSSION TITLE
development setup

### DISCUSSION THREAD

**Discussion by enricofer** [koordinates/kart#871] [Q&A]
I would like to submit some simple improvements like this https://github.com/koordinates/kart/compare/master...enricofer:kart:master
but I can't understand how to buildup a development install to easily develop and test new features or perform bug fixing. Documentation at this point seems to be a bit outdated. Could you provide some simple steps to start contribute to your project?

**Comment by rcoup** [comment]
Hi @enricofer — have a look at https://github.com/koordinates/kart/blob/master/CONTRIBUTING.md

Once you've got up to running `build/kart --version` successfully (or `.\build\venv\Scripts\kart.exe --version` on Windows) then you have a working development environment. From there any changes you make to the python code in the Kart source will work "live". We can definitely make that clearer in the docs 😄 

Feel free to keep asking questions though!
